specs/unit/validations/order-helpers.ts

```diff
@@ -57,3 +57,17 @@ function createValidOrderItems(): OrderItemValidationObject[] {
         options: []
     }];
 }
+
+
+export function createPotatoSkinSide(): OrderItemValidationObject {
```

Comment:
I don't think this function belongs in here. Functions should be in here that are required by several test files. This is very specific to one test file.